hierarchicalforecast/methods.py

```diff
@@ -629,39 +753,7 @@ def __init__(self,
         self.nonnegative = nonnegative
         self.insample = False
 
-    def reconcile(self,
```

Comment:
I have not seen the documentation, but will `showdoc(OptimalCombination.fit)` work as previous?
Or will it have `MinTrace`